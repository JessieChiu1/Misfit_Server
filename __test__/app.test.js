const request = require("supertest")
const app = require("../src/app")

describe("app.js", () => {
    it("response correctly when processing the home route", async() => {
        //act
        const response = await request(app).get("/")
        // expectation
        expect(response.status).toBe(200)
        expect(response.text).toBe("okay")
        expect(response.text).toMatchSnapshot()
    })
})