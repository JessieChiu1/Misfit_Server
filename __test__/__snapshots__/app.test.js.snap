// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app.js response correctly when processing the home route 1`] = `"okay"`;
